### PR TITLE
Adds colored stripes to departmental protolathes/techfabs

### PR DESCRIPTION
**Pull request by Wallemations**
## About The Pull Request
Gives protolathes/techfabs colored stripes to make it easier to tell them apart if they're moved for whatever reason.

Colors are all hex so they're easily editable.

Non-department protolathes/techfabs like the ones you find off-station remain unchanged.

## Why It's Good For The Game
Departmental individuality is good!!!!

## Changelog

:cl: Wallem
imageadd: Departmental protolathes/techfabs have colored stripes to tell the departments apart.
/:cl:

### CODE REVIEW

File: code/modules/research/machinery/_production.dm [hunk 1]
```diff
@@ -2,19 +2,27 @@
...
 	var/list/datum/design/matching_designs
-	var/department_tag = "Unidentified" //used for material distribution among other things.
+	/// Used for material distribution among other things.
+	var/department_tag = "Unidentified"
 
 	var/screen = RESEARCH_FABRICATOR_SCREEN_MAIN
 	var/selected_category
 
+	/// Does this production machine have a stripe?
+	var/has_color_stripe = FALSE
```
Comment (Mothblocks) [inline code/modules/research/machinery/_production.dm:22]: You don't need this, just check if `stripe_color` is null, and make the default null.

**Comment by Bawhoppen** [comment]
Hey I think Kor wanted these when they first came out 👍

File: code/modules/research/machinery/_production.dm (hunk 1 above)
Comment (Wallemations) [reply code/modules/research/machinery/_production.dm:22]: that's fair, I'll do that

Comment (Wallemations) [reply code/modules/research/machinery/_production.dm]: done

**Comment by Wallemations** [comment]
> Hey I think Kor wanted these when they first came out 👍

damn someone could'a told me, this is a port from a personal server.
Although on that one I didn't know how overlays worked so they're hard-sprited in instead of being a single greyscale mutable

**Review by Twaticus — APPROVED** [review]

**Pull request merged**